File: preview/src/main/java/com/guet/flexbox/preview/MockService.java
package com.guet.flexbox.preview;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.GET;

public interface MockService {
    @GET("/data")
    Call<Map<String, Object>> data();

    @GET("/layout")
    Call<byte[]> layout();
}

File: playground/src/main/java/com/guet/flexbox/playground/MockService.kt
package com.guet.flexbox.playground

import com.guet.flexbox.TemplateNode
import retrofit2.Call
import retrofit2.http.GET

interface MockService {
    @GET("/datasource")
    fun data(): Call<Map<String, Any>>

    @GET("/template")
    fun layout(): Call<TemplateNode>
}